executor/show.go

```diff
@@ -623,6 +623,10 @@ func (e *ShowExec) fetchShowCreateTable() error {
 		}
 	}
 
+	if tb.Meta().ShardRowIDBits > 0 {
+		buf.WriteString(fmt.Sprintf(" SHARD_ROW_ID_BITS=%d", tb.Meta().ShardRowIDBits))
```

Comment:
We should also use special-comment for `show create table`. Or users will meet error when migrating data from TiDB to MySQL.